prow/github/client.go

```diff
@@ -119,6 +119,7 @@ type IssueClient interface {
 	ListOpenIssues(org, repo string) ([]Issue, error)
 	GetIssue(org, repo string, number int) (*Issue, error)
 	EditIssue(org, repo string, number int, issue *Issue) (*Issue, error)
+	TransferIssue(org, dstRepoNodeID string, issueNodeID string) (*TransferIssueMutation, error)
```

Comment:
I don't think this should be part of the client

Reply:
Should it be part of the top level `Client` instead of the `IssueClient`? If it's not part of the GitHub interface I'd need to also remove it from mine here https://github.com/kubernetes/test-infra/pull/23153/files#diff-57052e6c3c1188cdb7b7d007f9a17201f1bec3747e55ea28ecfedc0e9306793aR25

Reply:
No, expose `MutateWithGitHubAppsSupport`, add the `TransferIssueMutation` to your plugin and call `MutateWithGitHubAppsSupport` there, similiar how we do it for `Query`